homeassistant/components/mqtt/device_tracker/__init__.py

```diff
@@ -4,6 +4,7 @@
 from homeassistant.components import device_tracker
 
 from ..mixins import warn_for_legacy_schema
+from .schema_discovery import PLATFORM_SCHEMA_MODERN  # noqa: F401
```

Comment:
We are importing this here so it can be imported transparently in `config_integration.py`